### ISSUE TITLE
Support for tauri

### ISSUE THREAD

**Issue by patrick91** [evoluhq/evolu#318]
Would it be possible to add support for tauri?

Tauri has a plugin for SQL: https://github.com/tauri-apps/tauri-plugin-sql which supports Sqlite :)

**Comment by steida** [comment]
@patrick91 Yes, it's possible, but I have tasks with higher priority right now.

**Comment by steida** [comment]
Tauri 2 looks very interesting! I will make an example once it's released.

**Comment by patrick91** [comment]
@steida for my side project I started using the SQL plugin (with Y.JS and soon Loro), maybe I'll check evolu again in future 😊

Thanks for wanting to help!

**Comment by steida** [comment]
@patrick91 Which SQL plugin(s), please? Thank you

**Comment by patrick91** [comment]
This one: https://github.com/tauri-apps/tauri-plugin-sql/tree/v2

I did have some performance issue, that I have solved by doing a custom uri scheme, like described here: https://github.com/tauri-apps/tauri/discussions/5597

My issue was that I was sending a "big" (0.5mb) Y.JS document and something in the serialisation/deserialisation process was slow (but it might have been fixed in the latest commits)
